.github/workflows/add-asana-pr-comment.yml

```
@@ -0,0 +1,17 @@
+name: Add PR link to Asana task
+
+on:
+  pull_request:
+    types: [opened]
+
+jobs:
+  sync:
```

I'd just rename this step, because it seems unrelated to the actual action being performed :)

```suggestion
  add-asana-comment:
    name: Add Asana comment
```

---

Good catch, thanks!